x-pack/plugins/actions/server/mocks.ts

```diff
@@ -4,4 +4,24 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-export { actionsClientMock } from './actions_client.mock';
+import { PluginSetupContract, PluginStartContract } from './plugin';
+
+const createSetupMock = () => {
+  const mock: jest.Mocked<PluginSetupContract> = {
+    registerType: jest.fn(),
+  };
+  return mock;
+};
+
+const createStartMock = () => {
+  const mock: jest.Mocked<PluginStartContract> = {
+    execute: jest.fn(),
+    getActionsClientWithRequest: jest.fn(),
```

Comment:
Can we have it return one of these instead? https://github.com/elastic/kibana/blob/335d784c0e513197bb37d7ab571f9dc8354dcbac/x-pack/plugins/actions/server/actions_client.mock.ts#L22-L24

Alternatively, or perhaps in addition, we could export that mock here as well, so folks who just want to test with an actionClient, and don't want to muck around with setup/start, can get one easily.  I think I saw similar with the mock logger - there was a cheap way to get a logger, or a more elaborate path through a mock logger service, kinda thing.

Reply:
Fair enough, I will implement the changes. Thanks for the +1 ideas!

Reply:
Fixed in https://github.com/elastic/kibana/pull/57002/commits/7d31ab7fc7a2dde131ea9236277605941d6b28fc.